op-guide/tidb-v3.0-upgrade-guide.md

```diff
@@ -0,0 +1,123 @@
+---
+title: TiDB 3.0 升级操作指南
+category: deployment
+---
+
+# TiDB 3.0 升级操作指南
+
+本文档适用于从 TiDB 2.0 版本（v2.0.1 及之后版本）或 TiDB 2.1 RC 版本升级到 TiDB 3.0 GA 版本。TiDB 3.0 版本兼容 Kafka 版本的 TiDB-Binlog 以及 集群模式的 TiDB-Binlog 。
+
+## 升级兼容性说明
+
+- loading
+
+## 注意事项
+
+- loading
```

Comment:
ditto

Reply:
I don't know who to write here or delete it.